src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  box-sizing: border-box;
+  border-radius: $border;
+  width: 200px;
+  background: $white;
+  box-shadow: 1px 1px 2px rgba(74, 102, 118, 0.3);
+  font-family: Roboto, Helvetica, sans-serif;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $title-color-value;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 auto 4px;
+  }
+  &__desc {
+    color: $desc-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+
+    &-value {
+      color: $title-color-value;
+      align-items: flex-end;
+      display: flex;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 16px;
+    margin-top: 24px;
```

Comment:
move  margin-top: 24px; to  &__review as  margin-bottom: 24px;
Try to use either margin-top or margin-bottom for vertical indents